### DISCUSSION TITLE
Older version of DSS or an option to turn off the 'only 1 frames will be stacked'

### DISCUSSION THREAD

**Discussion by TheuTrich** [deepskystacker/DSS#211] [General]
I take >2000 RAW images of stars and can see more than what the DSS reported. Take a look at the image below(the resolution is scaled down to half of the original)
![IMG_20240822_214603](https://github.com/user-attachments/assets/8f89e393-cb84-4106-a50d-14d23fa0453a)
_(a bit cloudy that day)_

Some time ago, I found the old version of DSS(3.3.4 or 3.3.6) that doesn't have this issue.
For now, I can't stack the images in DSS with the 'only 1 is stacked'. Thanks
Sorry for not being able to get on and read this regularly.

**Comment by perdrix52** [comment]
Please follow instructions and report your problem on the mailing list at groups.io.

As for this image, it is so poor that I'm not surprised DSS is refusing to stack - only 9 obvious stare like objects and out of focus / blurred by clouds
